res/css/views/settings/_LayoutSwitcher.pcss

```diff
@@ -71,7 +72,7 @@ limitations under the License.
     }
 
     .mx_StyledRadioButton_checked {
-        background-color: rgba($accent, 0.08);
+        background-color: $accent-200;
```

Comment:
This will break custom themes e.g. https://github.com/aaronraimist/element-themes